apps/angular/projection/src/app/component/teacher-card/teacher-card.component.ts

```diff
@@ -1,35 +1,56 @@
+import { AsyncPipe } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
-import { FakeHttpService } from '../../data-access/fake-http.service';
+import {
+  FakeHttpService,
+  randTeacher,
+} from '../../data-access/fake-http.service';
 import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
-import { Teacher } from '../../model/teacher.model';
 import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-teacher-card',
-  template: `<app-card
-    [list]="teachers"
-    [type]="cardType"
-    customClass="bg-light-red"></app-card>`,
+  template: `<app-card [items]="teachers$ | async" class="bg-light-red">
+    <img cardImage src="assets/img/teacher.png" width="200px" />
+
+    <ng-template #listItemRef let-teacher>
+      <app-list-item (deleteEvent)="deleteTeacher(teacher.id)">
+        {{ teacher.firstname }}
+      </app-list-item>
+    </ng-template>
+
+    <ng-container ngProjectAs="[actions]">
```

Comment:
Why did you wrap it inside a ng-container ? juste add actions to all button that you want to project and it will work. One nesting less